env.example

```diff
@@ -66,3 +66,6 @@ export DEPLOY_VERSION=v3.11
 
 # Path to the request to send to the OSA RP.
 #export MANIFEST=test/manifests/normal/create.yaml
+
+# install etcd backup data from this storage blob (don't include the etcd/ container)
+#export RECOVER_ETCD_FROM_BACKUP=backup-foo
```

Comment:
don't believe this makes sense - as mentioned below, suggest creating a small dumb freestanding harness under cmd/ to call RecoverEtcdCluster. Once the admin API exists and the fake RP supports it, we can roll it in.

Reply:
Agree. We need to think how we will do this in AdminAPI world.